File: Software Technologies/CatShop/js/models/cat.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Cat = sequelize.define("Cat",{
        name: {
            type: Sequelize.TEXT,
            required: true,
            allowNull: false
        },
        nickname: {
            type: Sequelize.TEXT,
            required: true,
            allowNull: false
        },
        price: {
            type: Sequelize.INTEGER,
            required: true,
            allowNull: false
        },

    },{
        timestamps:false
    });

    return Cat;
};

File: C#/03.Software Technologies/CatShop/csharp/CatShop/Models/Cat.cs
namespace CatShop.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Cat
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public string Nickname { get; set; }
        [Required]
        public double Price { get; set; }


    }
}
